contents/handbook/people/share-options.md

```diff
@@ -32,19 +33,32 @@ These can be financially very lucrative, because PostHog will give you the oppor
  
 ### What does it mean to 'exercise' a share option?
  
-This simply means you decide to buy the shares at the price agreed in your option agreement. You should be careful here, as you may have personal tax implications in doing this. For this reason, most people only do this at the time of an exit event, i.e. sale to another company or going public. 
+This simply means you decide to buy the shares at the price agreed in your option agreement. The price you pay is called the 'exercise price' or 'strike price' - both terms are widely used, but they mean the same thing here.  
+
+You should be careful here, as you may have personal tax implications in doing this. For this reason, most people only do this at the time of an exit event, i.e. sale to another company or going public. 
+
 
 ### What are my share options actually worth?
 
 You can use [this handy calculator](https://docs.google.com/spreadsheets/d/1mj-5Uy3IB5I09ml-zFH9VXxssqfGUgif1SURcsTFZRU/edit?usp=sharing) to find out, as well as to model what they might be worth in the future. You'll need to make a copy first, and be signed in with your PostHog email address. 
  
+
 ### What if I leave PostHog before this exit event happens?
  
-Luckily we have set up terms that are industry-leading in their friendliness to team members! If you leave PostHog, you will have until June 2031 to exercise them. This date applies to share options granted in June 2021 - in future, there will probably be different dates for different grants which we will list out here.  
+Happily, we have set up terms that are industry-leading in their friendliness to team members! If you leave PostHog, you will have until June 2031 to exercise them. This date applies to share options granted in June 2021 - in future, there will probably be different dates for different grants which we will list out here.  
```

Comment:
Should it be ten years from everyone's grant date or the date their vesting starts?

Reply:
Oh yes you're right - that's left over from when we first did the share option grants. I'll update as we've done a couple of grants since then.